### PR TITLE
Add LingBot Support to Segmentation Tasks (Semantic, Panoptic, Instance)

### PR DESCRIPTION
**Pull request by mrpositron**
## What has changed and why?

Adds LingBot Vision backbone support to the DINOv3 EoMT **segmentation** task models, semantic, panoptic, and instance, mirroring the existing EUPE integration.

LingBot Vision backbones (a DINOv3 derivative, pretrained with masked boundary modeling) were already integrated as pretraining/distillation backbones (`dinov3/vit{s,b,l}16-lingbot`) and for linear semantic segmentation, but they were not selectable for the EoMT segmentation heads. This PR closes that gap so users can fine-tune EoMT segmentation models on top of LingBot weights.

LingBot ships three sizes (S/B/L), so each task registers exactly those:
- `dinov3/vits16-lingbot-eomt`
- `dinov3/vitb16-lingbot-eomt`
- `dinov3/vitl16-lingbot-eomt`

Changes:
- **Configs**: new registry entries in the semantic, panoptic, and instance EoMT config files. Panoptic entries set `num_joint_blocks` to match the corresponding plain DINOv3 sizes (3 for S/B, 4 for L).
- **Docs**: added the S/B/L LingBot models to the model lists in `semantic_segmentation.md`, `panoptic_segmentation.md`, and `instance_segmentation.md`.
- **Tests**: extended `list_model_names` / `is_supported_model` assertions in all three EoMT segmentation test files.

## How has it been tested?

- Verified all 9 new registry entries resolve to the correct backbone (`dinov3/vit{s,b,l}16-lingbot`), and that panoptic entries carry the correct `num_joint_blocks` (3/3/4 for S/B/L).
- Ran the three EoMT segmentation task-model test suites:
```
pytest tests/_task_models/dinov3_eomt_semantic_segmentation/test_task_model.py \
       tests/_task_models/dinov3_eomt_panoptic_segmentation/test_task_model.py \
       tests/_task_models/dinov3_eomt_instance_segmentation/test_task_model.py
# 27 passed, 1 xfailed
```
## Did you update [CHANGELOG.md](../CHANGELOG.md)?

- [x] Yes
- [ ] Not needed (internal change)

## Did you update the documentation?

- [x] Yes
- [ ] Not needed (internal change without effects for user)

### CODE REVIEW

**Review by liopeer — APPROVED** [review]
LGTM!
